### ISSUE TITLE
benchmark atoma-infer vs vllm

### ISSUE THREAD

**Issue by jorgeantonio21** [atoma-network/atoma-infer#122]

**Comment by jorgeantonio21** [comment]
Superseded by #164, which owns the atoma-vs-vLLM baseline table as the rung-0 exit evidence. The instrument is `crates/bench`, merged in PR #172; the protocol and procedure are in `docs/benchmarks/rung0-baseline.md` and the table renders to `docs/benchmarks/rung0-baseline-table.md`.

No measurement has been taken yet — that is the open half of #164 and waits on an H100 stint.
